packages/apps/docs/src/components/Blog/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,24 @@
+import { avatar } from './styles.css';
+
+import React, { FC } from 'react';
+
+interface IProps {
+  name?: string;
+}
+
+export const Avatar: FC<IProps> = ({ name }) => {
+  if (!name) return null;
+
+  const getInitials = (name: string): string => {
```

Comment:
This method could be moved outside of the react component, also it seems a bit naive in handling middle names. Are you sure the implementation is desired as is?